tests/json-fixtures/test_blockchain.py

```diff
@@ -226,7 +226,7 @@ def test_blockchain_fixtures(fixture_data, fixture):
     try:
         chain = new_chain_from_fixture(fixture)
     except ValueError as e:
-        raise AssertionError("could not load chain for %r" % fixture_data) from e
+        raise AssertionError("could not load chain for %r" % (fixture_data,)) from e
```

Comment:
*if* you get back in here this might as well be updated to use `.format(...)` to be consistent with the rest of the codebase.

Reply:
Yeah, that's probably better. I'll change it to use `format()`